README.md

```diff
@@ -1,17 +1,26 @@
 # @nxworker/\* monorepo
 
-Monorepo for the `nxworker` NPM organization. Plugins and utilities for Nx workspaces.
+Monorepo for the `@nxworker` NPM organization. Plugins and utilities for Nx workspaces.
 
-## Wallaby.js
+## Sponsors
```

Comment:
Should we apply to the sponsor program of GitHub?

![Screen Shot 2021-01-13 at 18 12 03](https://user-images.githubusercontent.com/9338604/104511071-f3626a00-55ca-11eb-82c5-d1b772a5c7fa.png)

Reply:
Would it benefit you? I don't want to deal with Danish taxes to earn a few bucks.

Reply:
I don't have any motivation related to this. It just seemed a logical question given the creation of the sponsor section.

Reply:
With these sponsors, we're getting useful gifts without having to deal with any paperwork 😄 Besides, we don't have any features to motivate sponsorship right now 😉